saturn_lockfree.opam.template

```diff
@@ -0,0 +1,3 @@
+pin-depends: [
+  [ "dscheck.dev" "git+https://github.com/ocaml-multicore/dscheck#c81177e2f17d8ca336e14b47251b4473b393e750" ]
```

Comment:
A new dscheck version must be released and this pin-depend removed before merging.